pom.xml

```
@@ -13,12 +13,12 @@
     <version>${revision}-${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <revision>6.8.1</revision>
+        <revision>6.9.0</revision>
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <jenkins.version>2.401.1</jenkins.version>
         <bom>2.401.x</bom>
-        <bom.version>2378.v3e03930028f2</bom.version>
+        <bom.version>2507.vcb_18c56b_f57c</bom.version>
```

Most recent release of bom-2.401.x
```suggestion
        <bom.version>2675.v1515e14da_7a_6</bom.version>
```